build/assets/custom-dictionary.txt

```diff
@@ -586,6 +592,7 @@ Priya
 Prolo
 proteogenomic
 pseudoalignment
+Pslam
```

Comment:
You might not need this anymore. I would not be surprised if `psalm` was in the dictionary.

```suggestion
```